install/kubernetes/cilium/templates/cilium-configmap.yaml

```diff
@@ -417,6 +417,12 @@ data:
   enable-ip-masq-agent: "true"
 {{- end }}
 
+{{- if hasKey .Values "wireguard" }}
+{{- if .Values.wireguard.enabled }}
```

Comment:
Maybe not in this PR, but I think we should have something like `--encryption=ipvs|wireguard` and also have this reflected in the helm charts.

Reply:
I agree this would be better.

Reply:
Created https://github.com/cilium/cilium/issues/15483 to address it before v1.10.